staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -551,7 +551,7 @@ func (p *patcher) patchResource(ctx context.Context, scope RequestScope) (runtim
 }
 
 func (p *patcher) prefixFromUserAgent() string {
-	return strings.Split(p.userAgent, "/")[0]
+	return prefixFromUserAgent(p.userAgent)
```

Comment:
I'd rather remove this `patcher` method and be consistent in all 3 locations

Reply:
Okay, after you suggested to do it as a method in the last PR i kept it, but agreed.